### DISCUSSION TITLE
Free Hosting

### DISCUSSION THREAD

**Discussion by kaaax0815** [coder/code-server#2591] [Q&A]
Are there any Hosting Provider that would fit the requirements and are free? And maybe even Lifetime (3 months would be enough)? I searched everywhere, but I couldn't find any good provider (euserv has bad dns) that works. If its a service, that starts like glitch on demand it would be fine, one things is that I cant provide a credit card (so google and aws doesnt work)

**Comment by nhooyr** [comment]
I'm not aware of anything other than the GCP or AWS free tiers.

**Reply by nhooyr** [reply]
But like you mentioned those require a valid CC.
